src/web/lib/components/AppHeader/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactSVG from 'react-svg';
+import { surveyUrl } from '../../../../lib/constants';
+
+import './index.scss';
+
+export const AppHeader = ({ hasExtension, appVersion = null }) => {
+  let survey = `${surveyUrl}?ref=app`;
+  if (appVersion) {
+    survey += `${survey}&ver=${appVersion}`;
+  }
+  return (
+    <div className="app-header">
+      <div className="app-header__content">
+        <div className="app-header__icon" />
+        <header>
+          <h1>Themer</h1>
+          <h3>A <a href="https://testpilot.firefox.com" target="_blank"
+           rel="noopener noreferrer">Firefox Test Pilot</a> Experiment</h3>
+        </header>
+      </div>
+      {!hasExtension &&
```

Comment:
Double checking #29:

> Survey link should only appear when the user has TRF installed

Should drop the `!` here - it's showing the survey link when the add-on is *not* installed.